students/uss_tomasz/lesson_06_dictionaries_tuples_sets_args_kwargs/6_6/args_sum.py

```diff
@@ -0,0 +1,8 @@
+from functools import reduce
+
+
+def do_sum(x1, x2):
+    return x1 + x2
+
+
+print(reduce(do_sum, [1, 2, 3, 4]))
```

Comment:
my proposal was to keep `sum_all` signature unchanged and use reduce internally like this:
```
def sum_all(*args):
  return reduce(lambda x, total: total + x, args)
```
Alternatively this could be solved with build-in `sum` function, which I'd suggest to use for production code.
```
def sum_all(*args):
  return sum(args)
```
Anyway, for exercise reason we can keep it as it is.